downstream/titles/hub/configuring-private-hub-rh-certified/master.adoc

```diff
@@ -4,14 +4,18 @@
 
 include::attributes/attributes.adoc[]
 
-You can sync Automation Hub to use the Red Hat Certified Collections available through your Ansible Automation Platform subscription or community collections available through Ansible Galaxy.
+You can sync Automation Hub to use the Red Hat Certified Collections available through your Ansible Automation Platform subscription or community collections available through Ansible Galaxy. 
 
 Your organization can access and curate into a unique set of collections from all Red Hat Certified content in the Automation Hub service hosted on cloud.redhat.com.
 
+You can also configure private automation hub for signing and publishing Ansible content collections tailored to meet your organization's unique needs.
+
```

Comment:
@thaumos - please review line 11 as an addition to the preface of this document and let me know your thoughts at your earliest convenience. As a reminder, you and I discussed adding something to the preface to help enhance the cohesiveness of the doc. We didn't want Chapter 3 to come across as something that was just casually added to the end of this document. Thanks in advance for your feedback.

Reply:
LGTM!